### PR TITLE
Update karate_race_lite.txt

### PR DESCRIPTION
**Pull request by sugaarK**
move the karate race lite tune to a slider tune for the 4.4 release... if you dont like it just use the 4.3 preset and ignore the errors

### CODE REVIEW

**Review by limonspb — APPROVED** [review]
i guess you don't need PID values since you use sliders,
can just add "simplified_tuning apply"

File: presets/4.4/tune/karate/karate_race_lite.txt [hunk 1]
```diff
@@ -16,7 +16,8 @@
 #$ DESCRIPTION:
 #$ DESCRIPTION: <br>
 #$ DESCRIPTION:
-#$ DESCRIPTION: This tune should be used with 48khz PWM settings or firmware and it 'SHOULD' work with other ultralite 5" frames eg the ET-5 but use at your own risk.
+#$ DESCRIPTION: This tune should be used with 48khz PWM settings or firmware and it 'SHOULD' work with other ultralite 5" frames eg the ET-5.
+#$ DESCRIPTION: The 4.4 version is slider based to make it easier to adjust  for personal taste but use at your own risk.
```
Comment (limonspb) [inline presets/4.4/tune/karate/karate_race_lite.txt:20]: Extra space before "for"

**Comment by sugaarK** [comment]
> i guess you don't need PID values since you use sliders,
> can just add "simplified_tuning apply"

I used the diff from the quad I forgot how simplified tuning works.. wana make the recommendation I can commit?

File: presets/4.4/tune/karate/karate_race_lite.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.4/tune/karate/karate_race_lite.txt]: fixed

File: presets/4.4/tune/karate/karate_race_lite.txt [hunk 2]
```diff
@@ -107,25 +111,12 @@ set tpa_breakpoint = 1250
 
 #$ OPTION BEGIN (UNCHECKED): Dynamic idle for 4s 2850-3000kv
     #dynamic idle for 4s 2850-3000kv
-   set dyn_idle_min_rpm = 50
-   set dyn_idle_p_gain = 35
+	set dyn_idle_min_rpm = 50
+	set dyn_idle_p_gain = 35
```
Comment (haslinghuis) [inline presets/4.4/tune/karate/karate_race_lite.txt:115]: Please don't use tab
```suggestion
    set dyn_idle_min_rpm = 50
    set dyn_idle_p_gain = 35
```
```suggestion
	set dyn_idle_min_rpm = 50
	set dyn_idle_p_gain = 35
```

Comment (sugaarK) [reply presets/4.4/tune/karate/karate_race_lite.txt:115]: I’m confused!! This is the old file I only changed the pid settings and as I pointed out to @limonspb  my editor is formatting stuff to it all looks fine..  when it’s not 😭 if I don’t use tab to indent stuff what should I use the space bar ?

Comment (haslinghuis) [reply presets/4.4/tune/karate/karate_race_lite.txt:115]: Depending on editor you are using there should be a setting to convert tabs into 4 spaces.

Comment (limonspb) [reply presets/4.4/tune/karate/karate_race_lite.txt:115]: > I didn’t rifle through the rest of the file sorry it’s a carry over from the old version

old file didn't have tabs :)

Comment (sugaarK) [reply presets/4.4/tune/karate/karate_race_lite.txt:115]: k now have my editor showing me what's going on

**Review by limonspb — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**